lib/agent0/README.md

```diff
@@ -1,62 +1,45 @@
+
+> **💡NOTE:**
+> 
+> **pyenv install for Linux:**
+> 
+> If you're running bash on Linux, then to install Pyenv you’ll want to follow the [automatic installer](https://github.com/pyenv/pyenv#automatic-installer) instructions, and then [add pyenv to your shell environment](https://github.com/pyenv/pyenv#set-up-your-shell-environment-for-pyenv).
+> 
+> **using a fork:**
+> 
+> If you want the ability to PR new agent policies into our repo then we recommend you make a fork of the delvtech/agent0 repo.
+> ([repo link](https://github.com/delvtech/agent0), [GitHub fork instructions](https://docs.github.com/en/get-started/quickstart/fork-a-repo?tool=webui&platform=mac)).
 
-Make a fork of the delvtech/agent0 repo. ([repo link](https://github.com/delvtech/agent0), [GitHub fork instructions](https://docs.github.com/en/get-started/quickstart/fork-a-repo?tool=webui&platform=mac)).
 
 1.2.
-Install agent0 packages by following the installation instructions found [on github](https://github.com/delvtech/agent0/blob/main/INSTALL.md).
-
-    >**💡NOTE:**
-    >
-    >**pyenv install tips:**
-    >
-    >If you're using an AWS server running bash on Linux, then to install Pyenv you’ll want to follow the [automatic installer](https://github.com/pyenv/pyenv#automatic-installer) instructions, and then [add pyenv to your shell environment](https://github.com/pyenv/pyenv#set-up-your-shell-environment-for-pyenv).
-    >
-    >**git clone the correct repo:**
-    >
-    >You’ll want to clone your fork,
-    >e.g.: `git clone https://github.com/[YOUR_USERNAME]/agent0.git agent0`
-    >
-    >You’ll probably also want to also configure the remote (`delvtech`) repository.
-    >Once you’ve made the fork and cloned it, navigate inside (`cd agent0`), and run
-    >
-    >```bash
-    >git remote add upstream git@github.com:delvtech/agent0.git
-    >```
-    >
-    >to sync up an `upstream` repository.
-    >Then whenever you want to sync your fork with the latest code you can run:
-    >
-    >```bash
-    >git fetch upstream
-    >git merge upstream/main
-    >```
-    >
-    >**Hyperdrive contracts:**
-    >
-    >You don’t need to do any of the optional install Hyperdrive steps for eth_agents to work.
-    >
-
-1.3.
-Navigate to the `agent0` folder: `cd agent0/`
-
-1.4.
-Run the agent0 tests to verify that everything installed correctly by executing `python -m pytest`. Make sure you have enabled the correct Python environment, and installed the `requirements-dev.txt` file first.
+Run the agent0 tests to verify that everything installed correctly by executing `python -m pytest` from the repository root. Make sure you have enabled the correct Python environment, and installed the `requirements-dev.txt` file first.
+
+
+## Interactive Hyperdrive
+
+You can run Interactive Hyperdrive directly once the libraries are installed. See our [example script](https://github.com/delvtech/agent0/blob/main/lib/agent0/examples/interactive_hyperdrive_example.py) to get started.
```

Comment:
I would add that interactive hyperdrive requires docker engine (I typically follow instructions posted in infra) and anvil (I typically follow instructions posed in hyperdrive) installed. They're both system wide installations and can't be installed via virtual environment (as far as I know).

Reply:
good point! I'll bring those install instructions over.